pkgs/servers/mlflow-server/default.nix

```diff
@@ -5,7 +5,7 @@ let
 in
 py.toPythonApplication
   (py.mlflow.overridePythonAttrs(old: rec {
-    pname = "mlflow-server";
+    # pname = "mlflow-server";
 
```

Comment:
```suggestion
```

Reply:
changing the pname caused the pythonRelaxDeps to not work. so simplest change is to keep the original pname.

Reply:
Yes, but it is not necessary to keep the comment here. Just remove the line.

Reply:
fixed.

Reply:
Not really.

Reply:
sorry.  forgot to push